docusaurus/docs/learn/quickstarts/network/local-no-docker.md

```diff
@@ -29,7 +32,7 @@ utility be installed):
 This script will perform an 'express' install of Ziti which does the following:
 
 * download the latest version of the Ziti components (`ziti`, `ziti-controller`, `ziti-edge-router`, `ziti-tunnel`)
-* extract the components into a predefined location: ~/.ziti/quickstart/$(hostname)
+* extract the components into a predefined location: ~/.ziti/quickstart/$(hostname -s)

```

Comment:
i don't see a need for `-s`. why ?

Reply:
For accuracy to match `ziti-cli-functions.sh`